NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java

```diff
@@ -139,9 +139,12 @@ static void loadComments() {
         c.addComment("settings.use-tcp-fastopen", "Options: 0 - Disabled.; 1 - TFO is enabled for outgoing connections (clients).; 2 - TFO is enabled for incoming connections (servers).; 3 - TFO is enabled for both clients and servers.");
         c.addComment("settings.tile-entity-ticking-time", "Ticking time (20 ticks per second) for usage on tile entity operations.");
         c.addComment("settings.item-dirty-ticks", "Controls the interval for the item-dirty check. Minecraft checks an item every tick to see if it was changed. This can be expensive because it also needs to check all NBT data. Spigot only checks for basic count/data/type data and does a deep check every 20 ticks by default.");
-        c.addComment("settings.use-tcp-fastopen", "Enables the TCP_FASTOPEN socket option");
+        c.addComment("settings.tcp-fastopen-mode", "Enables the TCP_FASTOPEN socket option");
```

Comment:
Please avoid modding already defined configs

Reply:
I wasn't modifying it, I was fixing the comment (https://github.com/CobbleSword/NachoSpigot/commit/3d837bc502594ab14e68f5323578ddc5b078cd6c#commitcomment-60598696), however I did this incorrectly, but it is fixed in https://github.com/CobbleSword/NachoSpigot/pull/269/commits/2efad981da97a3e5bfbeb294bdf4d715f8fdf98e.